packages/manager/apps/hycu/src/components/DeleteModal/__snapshots__/deleteModal.spec.tsx.snap

```diff
@@ -0,0 +1,73 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
```

Comment:
I am not fan of snapshot on unit test. For me it's as say : "I don't known what I want and what it's important , so I don't known what I test"